### ISSUE TITLE
[bitnami/sonarqube] volumePermissions should probably run after custom init containers

### ISSUE THREAD

**Issue by pathob** [bitnami/charts#9525]
### Name and Version

bitnami/sonarqube 1.0.3

### What is the problem this feature will solve?

Dear Bitnami team,

I'm currently evaluating your SonarQube chart and trying to integrate a custom init container that downloads SonarQube plugins from a Maven repository. It looks like this custom init container is leaving file permissions behind, that make it unable for SonarQube to start. As far as I understand, the `volumePermissions` feature is meant to fix such permission issues. And then, it would probably make sense, to run this container after the custom init containers.

### What is the feature you are proposing to solve the problem?

Make volumePermissions run after the custom init containers, I can provide a PR.

### What alternatives have you considered?

The init container shouldn't mess up permissions, but this is sometimes hard to fix.

**Comment by carrodher** [comment]
Thanks for reporting this issue, yes, it makes sense to run the `volumePermissions` init container after others.
Would you like to contribute by creating a PR to solve the issue? The Bitnami team will be happy to review it and provide feedback. [Here](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md) you can find the contributing guidelines.

**Comment by hectoralicea** [comment]
> I'm currently evaluating your SonarQube chart and trying to integrate a custom init container that downloads SonarQube plugins from a Maven repository. It looks like this custom init container is leaving file permissions behind, that make it unable for SonarQube to start. As far as I understand, the `volumePermissions` feature is meant to fix such permission issues. And then, it would probably make sense, to run this container after the custom init containers.

Did you ever get this working?  Trying to use this feature today.

**Comment by pathob** [comment]
Unfortunately not, my suggestion wasn't really working and at the end we decided for the official Sonarqube chart even though it's conceptionally really bad at some points, e.g. at setting the admin password.

We solved the issues with the official vendor chart with Helmfile and its "ad-hoc dependency" feature. With the ad-hoc dependencies we can inject some custom stuff into the Helm chart, e.g. a database script that now sets the admin credentials in a declarative way.

Maybe that helps as I guess there is a specific reason why you looked into the Bitnami chart after all. :)
